packages/docs/getting-started/running-attestation-service.md

```diff
@@ -7,26 +7,27 @@
-| DATABASE_URL            | The URL under which your database is accessible, currently supported are `postgres://`, `mysql://` and `sqlite://`                                                     |  |
-| CELO_PROVIDER           | The URL under which a celo blockchain node is reachable, i.e. something like `https://integration-forno.celo-testnet.org`                                              |  |
-| ACCOUNT_ADDRESS         | The address of the validator account                                                                                                                                   |  |
-| ATTESTATION_PRIVATE_KEY | The private key with which attestations should be signed. You could use your account key for attestations, but really you should authorize a dedicated attestation key |  |
-| APP_SIGNATURE           | The hash with which clients can auto-read SMS messages on android                                                                                                      |  |
-| SMS_PROVIDERS           | A comma-separated list of providers you want to configure, we currently support `nexmo` & `twilio`                                                                     |  |
+| Variable                   | Explanation                                                                                                        |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| DATABASE_URL               | The URL under which your database is accessible, currently supported are `postgres://`, `mysql://` and `sqlite://` |  |
+| CELO_PROVIDER              | The URL under which a celo blockchain node is reachable and the attestation signer is unlocked                     |  |
+| ACCOUNT_ADDRESS            | The address of the validator account                                                                               |  |
+| ATTESTATION_SIGNER_ADDRESS | The address of the attestation signer that was authorized by the validator account                                 |  |
+| APP_SIGNATURE              | The hash with which clients can auto-read SMS messages on android                                                  |  |
+| SMS_PROVIDERS              | A comma-separated list of providers you want to configure, we currently support `nexmo` & `twilio`                 |  |
 
 A part of that we are going to setup the following environment variable about the Attestation Service Docker image:
 
 ```bash
-export CELO_IMAGE_ATTESTATION="us.gcr.io/celo-testnet/celo-monorepo@sha256:708ea8b24736a755c4dd3792e8973a6f0bf92b1f91edceb8e0b603ad66f2d70c"
+export CELO_IMAGE_ATTESTATION="us.gcr.io/celo-testnet/celo-monorepo:attestation-service-708ea8b24736a755c4dd3792e8973a6f0bf92b1f91edceb8e0b603ad66f2d70c"
```

Comment:
Tag should be `baklava`

Reply:
Will do once attestation service related changes have made it in